src/utils/middleware/helper/company.authentication.helper.ts

```diff
@@ -13,4 +14,8 @@ export class CompanyAuthenticationHelper {
         return companyNumber;
     }
 
+    static getCompanyNumberFromExtraData(req: Request): string | undefined {
```

Comment:
I think this should be in session.ts, with the other getters, and then you can use, getRequiredValue